tester.py

```diff
@@ -72,6 +72,22 @@ def dnanexus_cli(url, filename, reference_name=None, start=None, end=None):
             subprocess.check_call(cmd, stdout=outfile)
 
 
+def sanger_cli(url, filename, reference_name=None, start=None, end=None):
+                                   
```

Comment:
Yes, it is public - here is the link - https://github.com/wtsi-npg/npg_ranger/blob/devel/bin/client.js